File: test_archives/py_tests/batch4/test_function9.py
from functions.function9 import function9
import pytest

def test_function9_1(): 
    assert function9('abc', 'def', 1, 2.3) == 9.3

def test_function9_2(): 
    assert function9('xyz', 'uvw', 5, 7.8) == 18.8

File: test_archives/cpp_tests/batch4/test_function9.cpp
#include "../catch.hpp"
#include "../functions/function9.cpp"

TEST_CASE( "Tests for function9", "[function9]" ) {
    REQUIRE( function9("abc", "def", 1, 2.3) == 9.3 );
    REQUIRE( function9("xyz", "uvw", 5, 7.8) == 18.8 );
}